packages/db/src/scripts/seed.ts

```
@@ -0,0 +1,145 @@
+/**
+ * Seed script to populate the database with sample data
+ *
+ * This creates:
+ * - 5 sample drivers with unique RFIDs
+ * - Sample telemetry packets for each driver using generateFakeTelemetryData() from shared
+ * - Sample lap data for each driver
+ */
+
+// Sample driver data
+const sampleDrivers = [
+  { rfid: "DRIVER001", Name: "Alex Johnson" },
```

ESLint

[error] Expected object keys to be in ascending order. 'Name' should be before 'rfid'.

(sort-keys/sort-keys-fix)